google/cloud/storage/internal/sign_blob_requests.cc

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/storage/internal/sign_blob_requests.h"
+#include "google/cloud/storage/internal/nljson.h"
+
+namespace google {
+namespace cloud {
+namespace storage {
+inline namespace STORAGE_CLIENT_NS {
+namespace internal {
+
+std::ostream& operator<<(std::ostream& os, SignBlobRequest const& r) {
+  auto join = [](std::vector<std::string> const& v) -> std::string {
```

Comment:
are we not ready to use absl StrJoin() here?

Reply:
Because we don't have Abseil at all in the project?